libs/designsystem/src/lib/components/form-field/form-field.component.ts

```diff
@@ -40,9 +41,8 @@ export class FormFieldComponent
 
   @ContentChild(InputCounterComponent, { static: false }) counter: InputCounterComponent;
   @ContentChild(RadioGroupComponent) private radioGroupComponent: RadioGroupComponent;
-  @ContentChild(RadioGroupComponent, { read: ElementRef }) private radioGroupElement: ElementRef<
-    HTMLElement
-  >;
+  @ContentChild(RadioGroupComponent, { read: ElementRef })
+  private radioGroupElement: ElementRef<HTMLElement>;
```

Comment:
Disclaimer: I'm no Angular expert 😎

Is this correct? Is separating `private radioGroupElement` from `@ContentChild` equivalent to having them on the same line?

Reply:
Yep! It's the same thing 😄 

Fun fact: Actually decorators are an experimental typescript thingy and not Angular specific. 
![image](https://user-images.githubusercontent.com/17269698/161001223-1ae75cfd-f991-48f1-9bd7-db111565ba01.png)